src/v/raft/state_machine_manager.cc

```diff
@@ -387,14 +387,17 @@ void state_machine_manager::maybe_start_background_apply(
         return entry->background_apply_mutex.get_units().then(
           [this, entry](auto u) {
               return ss::with_scheduling_group(
-                       _apply_sg,
-                       [this, entry] { return background_apply_fiber(entry); })
-                .finally([u = std::move(u)] {});
+                _apply_sg, [this, entry, u = std::move(u)]() mutable {
+                    return background_apply_fiber(entry, std::move(u)).then([] {
+                        return ss::sleep(1s);
```

Comment:
is this sleep intentional or a for simulating the race?

Reply:
oh, i need to remove it, left from debugging

Reply:
removed